src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheckTest.java

```diff
@@ -270,6 +305,7 @@ public void testStaticGroupsAlphabeticalOrder() throws Exception {
         checkConfig.addAttribute("option", "top");
         checkConfig.addAttribute("groups", "org, java");
         checkConfig.addAttribute("sortStaticImportsAlphabetically", "true");
+        checkConfig.addAttribute("separated", "true");
```

Comment:
@romani Are you ok with adding all these properties to existing configs?
It seems like we are losing tests for `separated` set to `false`.

Reply:
good catch.

@djydewang , remove such changes. If you need more tests - create new.

Reply:
＠romani I didn'd add all these properties to existing configs.And there are many configs without attribute 'separated'.So we don't lose tests for sparated to false.

Reply:
default value for separated is FALSE - http://checkstyle.sourceforge.net/config_imports.html#ImportOrder
as you make it TRUE , you change meaning of UT.
If scope of your issue it is not required.

Reply:
> default value for separated is FALSE -
Right,these UT's task are not to test extra separated,So I make it true.

Reply:
please revert all such changes. 
Do not change other UTs meaning, all combinations or options/properties are ctitical in testing.

Reply:
Get it.